### ISSUE TITLE
can't read file when prefix a site.property

### ISSUE THREAD

**Issue by scsskid** [brob/eleventy-plugin-svg-contents#2]
Hi,
great plugin!

plugin works when I hardcode the path, but when using like this it can't read the file. thanks for an idea

*.njk
```
{{ site.dirs.icons + 'icon-github.svg' | svgContents | safe }}
```

site.js
```
{
  dirs: {
    icons: '/src/assets/images/icons/'
  }
}
```

**Comment by brob** [comment]
Interesting! I'll take a look at this. 

Sorry for the delay in response, my notifications apparently weren't set up on this repo at all. Have you worked around this in the past 6 months? 

Either way, I'll try to take a look into this in the next couple of weeks (after I merge in a big PR which should make working on this easier).

Thanks for letting me know!

**Comment by brob** [comment]
I haven't found a definitive answer yet, but here's what I'm assuming based on my testing:

1. This works if you use a set tag, such as `{% set path = site.dirs.icons + "simple.svg" %}` 
2. When I render `{{ site.dirs.icons + "simple.svg"}}` I get the expected string.
3. When I look at the error, it's ignoring the variable expression completely.
4. I believe the filters are applied to each piece of the expression separately, instead of after the concatenation.

In other words, I believe that with the way Nunjucks renders expressions it would render `{{ site.dirs.icons }}` then render `{{ "simple.svg" | svgContent | safe  }}` and THEN try to concat those two together. I'll research more, but I think this is the case and it's probably not something the plugin can overcome. I'm betting the end solution will be to use a `{% set %}` tag which I know may not be ideal.

**Comment by brob** [comment]
And as soon as I send that, I get a message from a community friend! Overall with the way the parser works, my assumption is correct that it's rendering "simple.svg" | svgContent as it's own expression for the order.

There's a simpler path than setting the variable though. If you wrap your concat in parenthesis, you'll evaluate that as one expression instead of two.

If you adjust your code to look like this, it should work:

```
{{ ( site.dirs.icons + 'icon-github.svg' ) | svgContents | safe }}
```

I don't think there's much the plugin can do to prevent this, so I'm going to close the issue out.
